Simplified/NYPLSecrets.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
```

Comment:
I just tried to add a random file already under git to `.gitignore`, but once I modify that file git will show me that that file has changes. So that's a problem. I think once it's under source control, .gitignore doesn't matter anymore.

Reply:
Try `git update-index --skip-worktree Simplified/NYPLSecrets.swift`
The line above should ignore any changes but still manage to keep the file in source control
Let me know if it works or not